sandbox/grist/functions/lookup.py

```diff
@@ -26,6 +26,18 @@ def COLUMNS(range):
   """Returns the number of columns in a specified array or range."""
   raise NotImplementedError()
 
+def FLATTEN_RECORDS_LIST(list_of_records, dedup=False):
+    """Returns a flat list of records"""
+    if type(list_of_records) is not list:
+        raise TypeError("Unexpected Argument, waiting of a list of Records")
+    if len(list_of_records) == 0:
+        return list_of_records
+    flatten_list = [el for val in list_of_records for el in val]
+    el_type = {el.__class__.__name__ for el in flatten_list}
+    if list(el_type)[0] != 'Record':
```

Comment:
Why do you have to check the first element only?

Wouldn't `'Record' in el_type` be more robust?